pkg/node/node_connectivity_checker.go

```diff
@@ -356,7 +363,7 @@ func (s *Service) populateTargetsInCache(array *array.PowerStoreArray) {
 				}
 				break
 			}
-			log.Debugf("Portal %s is not rechable from the node", address.Portal)
+			log.Debugf("Portal is not rechable from the node")
```

Comment:
@karthikk92 Could you print 'address.Portal' here.? It will be helpful in debugging.